deployments/compose/notebook/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/bitnami/spark:3
+USER root
+RUN useradd -ms /bin/bash notebook
+USER notebook
+WORKDIR /home/notebook
+ENV HOME /home/notebook
+ENV PATH ${PATH}:/home/notebook/.local/bin
+RUN pip install --user notebook findspark pyspark-kernel spylon-kernel
+COPY notebook-config.py /home/notebook/.jupyter/jupyter_notebook_config.py
+RUN mkdir /home/notebook/notebooks
+ENTRYPOINT []
+CMD ["jupyter", "notebook"]
```

Comment:
Why did you choose not to use an existing one from docker-hub Such as:
https://hub.docker.com/r/jupyter/all-spark-notebook
https://hub.docker.com/r/jupyter/pyspark-noteboo

Reply:
At this point I wanted to use the same base image with the same spark that we currently have in out bagel.

Reply:
If the out of the box experience will lake something - I agree we should move to another solution.